GPU/Common/SplineCommon.cpp

```diff
@@ -361,16 +383,45 @@ static void _SplinePatchFullQuality(u8 *&dest, u16 *indices, int &count, const S
 	}
 }
 
+template <bool origNrm, bool origCol>
+static inline void SplinePatchFullQualityDispatch3(u8 *&dest, u16 *indices, int &count, const SplinePatchLocal &spatch, u32 origVertType, int quality, int maxVertices) {
+	bool origTc = (origVertType & GE_VTYPE_TC_MASK) != 0;
```

Comment:
I found the technique in Wikipedia that is called "template meta programming". Actually I didn't know this template technique that unroll loops in compiling.
This way is a bit faster in release mode but the codes seemed a little confused for me, but I understood now :)
However, spline/bezier codes seem still difficulty so I want to rewrite more clear.
Well, I appreciate your work to fix spline/bezier completely, and highly improved performance.